T4MVCHostMvcApp/T4MVC Files/T4MVC.tt

```diff
@@ -751,7 +752,11 @@ void ProcessControllerType(CodeClass2 type, AreaInfo area, DateTime controllerLa
     if (type.IsAbstract)
     {
         // If it's abstract, set a flag and don't process action methods (derived classes will)
-        target.IsAbstract = true;
+        if (!HasControllerAttribute(type))
+            target.IsAbstract = true;
+        // ...unless it has the [T4MVC] attribute, then process all the action methods in the controller
+        else
+          ProcessControllerActionMethods(target, type);
```

Comment:
The style we use in T4MVC is 4 spaces, while you have 2 in a bunch of places (in this file and others).